src/components/Link/Link.styles.tsx

```
@@ -0,0 +1,36 @@
+        !['backgroundColor', 'linkSize', 'borderRadius'].includes(
+            prop.toString(),
+        ),
+})<StyledLinkProps>(
+    ({
+        theme: { spacing, mixins },
+        linkSize,
+        backgroundColor,
+        borderRadius,
+    }) => ({
+        ...mixins.flex('row', 'flex-start', 'center'),
+        padding: spacing(LinkSizeMap[linkSize!]),
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)